pkg/metricutil/metricutil.go

```diff
@@ -60,12 +61,12 @@ func camelCaseToSnakeCase(str string) string {
 
 // prometheusPushClient pushs metrics to Prometheus Pushgateway.
 func prometheusPushClient(job, addr string, interval time.Duration) {
+	pusher := push.New(addr, job)
+	pusher = pusher.Gatherer(prometheus.DefaultGatherer)
+	pusher = pusher.Grouping("instance", instanceName())
```

Comment:
Would you like to combine these lines together to use a chain-style like:
`pusher := push.New(addr, job).Gatherer(prometheus.DefaultGatherer).Grouping(...)`